src/System.Net.Security/src/System/Net/StreamFramer.cs

```diff
@@ -214,12 +216,13 @@ private void ReadFrameComplete(IAsyncResult transportResult)
                 int bytesRead = _transportAPM.EndRead(transportResult);
                 workerResult.Offset += bytesRead;
 
-                if ((workerResult.Offset <= workerResult.End))
+                if (!(workerResult.Offset <= workerResult.End))
```

Comment:
>See above regarding diffing with Desktop.

I agree with this. We are trying very hard to reconcile two codebase (Core and Desktop) and re-align our tests as well. So, for now, keeping things as close as possible (without violating CoreFx coding guidelines) helps.